project/src/main/resource-cache.gd

```diff
@@ -27,9 +27,18 @@ export (bool) var verbose := false
 export (bool) var minimal_resources := false
 
 # maintains references to all resources to prevent them from being cleaned up
+# key: resource path
+# value: resource
 var _cache := {}
 var _cache_mutex := Mutex.new()
 
+# stores parsed version of json resources to speed up retrieval
```

Comment:
Comment should probably mention something about "Aseprite" or "regions". This isn't generic enough to apply to all json resources.

Reply:
Changed to `...versions of Aseprite json resources...`